modules/ossm-rn-new-features.adoc

```diff
@@ -228,6 +251,31 @@ spec:
 * {SMProductShortName} on ARM64 architecture is not supported.
 * OpenTelemetry API remains a Technology Preview feature.
 
+== New features {SMProductName} version 2.3.8
+//Update with 2.4.4
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
```

Comment:
Same question as above I'm just not sure where 4.10 support ended so I wanted to point this out.

Reply:
For OSSM, according to the posted policy, support for 4.10 ends with Release of 2.5 + 1 month. 2.5 has not been released yet so technically OSSM supports 4.10.

Reply:
No worries, the more you know. Just doing my due diligence. Thanks.